shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/ral/RALBackendHandlerFactory.java

```diff
@@ -39,14 +43,18 @@
      * @param sqlStatement RAL statement
      * @param backendConnection backend connection
      * @return RAL backend handler
+     * @throws SQLException SQL exception
      */
-    public static TextProtocolBackendHandler newInstance(final RALStatement sqlStatement, final BackendConnection backendConnection) {
+    public static TextProtocolBackendHandler newInstance(final RALStatement sqlStatement, final BackendConnection backendConnection) throws SQLException {
         if (sqlStatement instanceof QueryableRALStatement) {
             return QueryableRALBackendHandlerFactory.newInstance((QueryableRALStatement) sqlStatement, backendConnection);
         }
         if (sqlStatement instanceof UpdatableRALStatement) {
             return UpdatableRALBackendHandlerFactory.newInstance((UpdatableRALStatement) sqlStatement);
         }
+        if (sqlStatement instanceof SCTLStatement) {
```

Comment:
`QueryableRALStatement`, `UpdatableRALStatement`, and `SCTLStatement` are in parallel level?